Application/ExternalDeps/AssignmentsApi/ProjectsMock.cs

```diff
@@ -0,0 +1,21 @@
+namespace Application.ExternalDeps.AssignmentsApi;
+
+public static class ProjectsMock
```

Comment:
What is the purpose of this class? I think we can just add the `static readonly ProjectsResponse ProjectsData` to the AssignmentsApi directly